README.md

```diff
@@ -304,11 +304,14 @@ crud.select('customers', {{'<=', 'age', 35}})
 You can iterate across a distributed space using the `crud.pairs` function.
 Its arguments are the same as [`crud.select`](#select) arguments,
 but negative `first` values aren't allowed.
+Also since it's possible to return only rows without metadata
```

Comment:
> only rows

I think that rows is a tuples, maybe it would be better to say just "User could pass `use_tomap` flag (`false` by default) to iterate over flat tuples or objects."

Reply:
Ok, just `User could pass use_tomap flag (false by default) to iterate over flat tuples or objects.`